source/_integrations/spotify.markdown

```diff
@@ -72,29 +72,45 @@ client_secret:
 <div class='note'>
 
   If you receive an `INVALID_CLIENT: Invalid redirect URI` error while trying to
-  authenticate with your Spotify account, make sure to check the Redirect URI in
+  authenticate with your Spotify account, check the Redirect URI in
   the address bar after adding the new integration. Compare this value with the
   Redirect URI defined in the Spotify Developer Portal.
 
 </div>
 
 ## Using multiple Spotify accounts
 
-The integration support multiple Spotify accounts at once. You don't need to
+This integration supports multiple Spotify accounts at once. You don't need to
 create another Spotify application in the Spotify Developer Portal and no
 modification to the `configuration.yaml` file is needed. Multiple Spotify
 accounts can be linked to a _single_ Spotify application.
 
-To add an additional Spotify account to Home Assistant, repeat _only_ the steps
-in the [Activating the Spotify integration](#activating-the-spotify-integration)
-section. However, make sure you go to the Spotify website and log out before
-you do so.
+To add an additional Spotify account to Home Assistant, go to the Spotify website and log out, then repeat _only_ the steps
+in the [Activating the Spotify integration](#activating-the-spotify-integration) section. 
 
-## URI Links for Playlists
+## Playing Spotify playlists
 
 You can send playlists to Spotify using the `"media_content_type": "playlist"`, which is part of the
-[media_player.play_media](/integrations/media_player/#service-media_playerplay_media) service.
-The `media_content_id` can be obtained from the Spotify desktop-app by clicking on the more options ("...") next to the album art picture, selecting "share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the Spotify phone and web app).
+[media_player.play_media](/integrations/media_player/#service-media_playerplay_media) service, for example:
+```yaml
```

Comment:
- A code block needs to be surrounded by an empty new line.
- There is an empty line in the code block itself that is not needed.
- This code sample is incomplete and thus not copy-pastable.
- `alias` is not required and should be removed.

Reply:
The code block should now be a complete script with an empty line before and no `alias`. I did not see an empty line in the code block.
Hopefully this is acceptable now.